tests/storage/test_transactions.py

```diff
@@ -57,8 +56,11 @@ def test_initial_set_transactions(self) -> None:
         self.get_success(d)
 
     def test_large_destination_retry(self) -> None:
+        max_retry_interval = (
```

Comment:
Would also be good to add a `_s`/`_ms` suffix to make it more clear what the expected value should be (related to https://github.com/matrix-org/synapse/pull/15754#discussion_r1260141031)